### PR TITLE
Update EXECUTE BOOSTED to not include implicit EXECUTE

### PR DESCRIPTION
**Pull request by Hunterness**
Documentation of https://github.com/neo-technology/neo4j/pull/15033.
Companion to https://github.com/neo-technology/neo4j-manual-modeling/pull/2748.

### CODE REVIEW

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 1]
```diff
@@ -367,11 +367,11 @@ The following commands are only executable against the `system` database:
...
-| <<access-control-execute-boosted-user-defined-function, EXECUTE BOOSTED FUNCTION>>                                      | DBMS                  | Determines whether the user can execute functions with elevated privileges.
-| <<access-control-execute-boosted-procedure, EXECUTE BOOSTED PROCEDURE>>                                      | DBMS                  | Determines whether the user can execute procedures with elevated privileges.
-| <<access-control-execute-user-defined-function, EXECUTE FUNCTION>>                                                       | DBMS                  | Determines whether the user can execute functions.
-| <<access-control-execute-procedure, EXECUTE PROCEDURE>>                                                     | DBMS                  | Determines whether the user can execute procedures.
+| <<access-control-admin-procedure, EXECUTE ADMIN PROCEDURE>>                           | DBMS  | Determines whether the user can execute admin procedures.
+| <<access-control-execute-boosted-user-defined-function, EXECUTE BOOSTED FUNCTION>>    | DBMS  | Determines whether the user get elevated privileges when executing functions.
```
Comment (HannesSandberg) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: should we use boosted instead of elevated here?

Comment (Hunterness) [reply cypher/cypher-docs/src/docs/dev/glossary.asciidoc]: We quite often uses `elevated` when describing it (it was also called that before I changed to not being able to execute)

**Review by HannesSandberg — APPROVED** [review]

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 2]
```diff
@@ -368,11 +368,11 @@ The following commands are only executable against the `system` database:
...
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]:
```suggestion
| <<access-control-execute-boosted-user-defined-function, EXECUTE BOOSTED FUNCTION>>    | DBMS  | Determines whether the user gets elevated privileges when executing functions.
```

File: cypher/cypher-docs/src/docs/dev/glossary.asciidoc [hunk 3]
```diff
@@ -368,11 +368,11 @@ The following commands are only executable against the `system` database:
...
+| <<access-control-execute-boosted-procedure, EXECUTE BOOSTED PROCEDURE>>               | DBMS  | Determines whether the user get elevated privileges when executing procedures.
```
Comment (davidoliverSP2) [inline cypher/cypher-docs/src/docs/dev/glossary.asciidoc]:
```suggestion
| <<access-control-execute-boosted-procedure, EXECUTE BOOSTED PROCEDURE>>               | DBMS  | Determines whether the user gets elevated privileges when executing procedures.
```

**Review by davidoliverSP2 — APPROVED** [review]
👍 Thanks @Hunterness

**Pull request merged**